### PR TITLE
[WIP] Update readme with instructions for using the browser from a container

### PR DESCRIPTION
**Pull request by jzoldak**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -141,17 +142,51 @@ Using the Browser from a Docker Container
...
+    * Set it to a value without spaces. "localhost" is a good one.
+    * This means passing adding to the docker command line: `--env no_proxy=localhost`
+
+* If you want to see the display of the running container, choose a container that has
+    VNC installed.
+
+    * Those are the ones that end in "debug"
+    * E.g. selenium/standalone-firefox-debug
+
+* I've been using this, which has FF 46 if I remember correctly:
```
Comment (raeeschachar) [inline README.rst]: Correction: Firefox 47

**Comment by edx-webhook** [comment]
Thanks for the pull request, @jzoldak! I've created [OSPR-2350](https://openedx.atlassian.net/browse/OSPR-2350) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by mduboseedx** [comment]
@raeeschachar Do you know if this PR is still worth pursuing?

**Comment by raeeschachar** [comment]
@mduboseedx I don't think it is needed anymore. e2e tests can run from devstack now and I think enough information is provided [here](https://github.com/edx/edx-e2e-tests)

**Pull request closed without merging**